apps/client/src/views/cuesheet/CuesheetPage.tsx

```diff
@@ -53,29 +53,44 @@ export default function CuesheetPage() {
         return;
       }
 
-      const previousValue = event.custom[accessor];
-
-      if (previousValue === payload) {
+      // is it a custom fields
+      if (typeof accessor === 'string' && accessor.startsWith('custom_')) {
+        const previousValue = event.custom[accessor];
+
+        // skip if there is no value change
+        if (previousValue === payload) {
+          return;
+        }
+
+        // check if value is valid
+        // in anticipation to different types of event here
+        if (typeof payload !== 'string') {
+          return;
+        }
```

Comment:
question: can you explain this guard? I am not sure what we expect here?

Reply:
left over from the original logic, I can clean it up a bit